GitUI/CommandsDialogs/Menus/ToolStripMenuItemEx.cs

```diff
@@ -14,6 +14,9 @@ internal abstract class ToolStripMenuItemEx : ToolStripMenuItem, ITranslate
         protected GitUICommands UICommands
             => (_getUICommands ?? throw new InvalidOperationException("The button is not initialized")).Invoke();
 
+        protected GitModule Module
+            => UICommands.Module;
```

Comment:
Is this used here, or somewhere else?
![image](https://user-images.githubusercontent.com/4403806/221345052-e36b613d-6ecb-43ad-b923-46ef0100bf8a.png)

Reply:
Yes, here. I expect more usages when we factor out more submenus. We use to have a `Module` property everywhere.